pkg/daemon/discover/discover.go

```diff
@@ -51,6 +55,7 @@ var (
 	cmName          string
 	cm              *v1.ConfigMap
 	udevEventPeriod = time.Duration(5) * time.Second
+	udevFilter      = []string{"rbd", "nbd", "dm-"}
```

Comment:
I'd suggest we just remove the `for` loop later on line 184 define this var here with the regex included:
`udevFilter = []string{"(?i)dm-[0-9]+", "(?i)rbd[0-9]+", "(?i)nbd[0-9]+"}`